quantus/helpers/normalise_func.py

```diff
@@ -1,15 +1,28 @@
 """This module provides some basic functionality to normalise and denormalise images."""
-from typing import Union
+from typing import Callable, Dict, Optional, Union
 import numpy as np
 
 
-def normalise_by_max(a: np.ndarray) -> np.ndarray:
-    """ "Normalize attributions by the maximum absolute value of the explanation."""
+def normalise_batch(
+        arr: np.ndarray,
+        normalise_func: Callable,
+        **normalise_func_kwargs,
+) -> None:
+    """Normalise complete batch by given normalisation function."""
+    # TODO: use something like np.apply_along_axis instead of for-loop
```

Comment:
I would suggest something like this:

1. Add an axes argument to all normalisation functions, to specify axes over which the normalisation is performed.
2. Use this argument to exclude the batch axis during normalisation

Reply:
+1